### PR TITLE
LPS-201206 [FE] Refactor empty node to use ClayCard

### PR DESCRIPTION
**Pull request by igor-franca**
related [ticket](https://liferay.atlassian.net/browse/LPS-201206)

### CODE REVIEW

**Comment by danvb112** [comment]
I have just start the review

File: modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModelBuilder/Diagram/Diagram.tsx [hunk 1]
```diff
@@ -227,11 +207,7 @@ function DiagramBuilder({
 				connectionMode={ConnectionMode.Loose}
 				edgeTypes={EDGE_TYPES}
 				elements={
-					!isLoadingObjectFolder
-						? elements.length
-							? elements
-							: emptyNode
-						: []
+					!isLoadingObjectFolder && !!elements.length ? elements : []
```
Comment (danvb112) [inline modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModelBuilder/Diagram/Diagram.tsx:210]: Hey @igor-franca, I'm trying to find a reason to make this entire condition, in my tests here it seems that nothing changes if we remove this entire condition, couldn't we just use elements?

```tsx
elements={elements}
```

**Review by danvb112 — CHANGES_REQUESTED** [review]
Hey @igor-franca, great job on this task, I have some suggestions :smile:

File: modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModelBuilder/Diagram/Diagram.tsx (hunk 1 above)
Comment (igor-franca) [reply modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModelBuilder/Diagram/Diagram.tsx:210]: Make sence to me! Thansk

File: modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModelBuilder/EmptyObjectFolderCard/EmptyObjectFolderCard.scss [hunk 2]
```diff
@@ -0,0 +1,29 @@
+.lfr-objects__model-builder-empty-object-folder-card {
+	&-body {
+		align-items: center;
+		display: flex;
+		flex-direction: column;
+		padding-top: 1rem !important;
+	}
+
+	&-container {
```
[the remark this answers is not in the document]
Comment (igor-franca) [reply modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModelBuilder/EmptyObjectFolderCard/EmptyObjectFolderCard.scss:9]: I applied the change and I think the result was very good. Thanks!

File: modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModelBuilder/EmptyObjectFolderCard/EmptyObjectFolderCard.tsx [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+interface EmptyObjectFolderCardProps {
+	setShowModal: (value: React.SetStateAction<ModelBuilderModals>) => void;
+}
+
+export default function EmptyObjectFolderCard({
+	setShowModal,
+}: EmptyObjectFolderCardProps) {
+	return (
+		<div className="lfr-objects__model-builder-empty-object-folder-card-container">
```
[the remark this answers is not in the document]
Comment (igor-franca) [reply modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModelBuilder/EmptyObjectFolderCard/EmptyObjectFolderCard.tsx]: Done!

File: modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModelBuilder/EmptyObjectFolderCard/EmptyObjectFolderCard.scss [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+		border: 2px solid #d3d6e0;
+		border-radius: 4px;
```
[the remark this answers is not in the document]
Comment (igor-franca) [reply modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModelBuilder/EmptyObjectFolderCard/EmptyObjectFolderCard.scss]: Done

**Comment by danvb112** [comment]
ci:test:sf

**Review by danvb112 — APPROVED** [review]
LGTM! :tada:

**Comment by igor-franca** [comment]
ci:test:relevant

**Comment by igor-franca** [comment]
ci:test:object

**Comment by igor-franca** [comment]
ci:forward:force

**Comment by igor-franca** [comment]
ci:forward:force

**Pull request closed without merging**